packages/core/src/components/Panel/Panel.vue

```diff
@@ -1,5 +1,5 @@
 <script lang="ts" setup>
-import type { PanelProps } from '../../types'
+import type { PanelProps } from '../../types/panel'
```

Comment:
without this change, the build failed with the following error after I added the non-setup `<script>` tag:

![image](https://user-images.githubusercontent.com/5778364/218651880-a1b0aff1-aa91-47ab-879f-b150a5f7b6f6.png)

I referenced [`Handle.vue`](https://github.com/bcakmakoglu/vue-flow/blob/7a6f37df0416d01eba30d05c2f933b9029be098d/packages/core/src/components/Handle/Handle.vue#L4) which also has a non-setup `<script>` block and this appeared to be the only difference between them; indeed, it fixed the build. I'm not familiar with `<script setup>`, though - so I'm not sure if this is the correct fix or not.